### DISCUSSION TITLE
Error Msg while running nuclei.

### DISCUSSION THREAD

**Discussion by qusaialhaddad** [projectdiscovery/nuclei#1590] [Q&A] [answered]
I am getting lot of these errors on each template. 

[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-17431.yaml': yaml: unmarshal errors:
  line 10: cannot unmarshal !!seq into string
  line 13: cannot unmarshal !!map into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-15138.yaml': yaml: unmarshal errors:
  line 8: cannot unmarshal !!seq into string
  line 11: cannot unmarshal !!map into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-6910.yaml': yaml: unmarshal errors:
  line 11: cannot unmarshal !!map into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-20824.yaml': yaml: unmarshal errors:
  line 10: cannot unmarshal !!map into string
  line 15: cannot unmarshal !!seq into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-12634.yaml': yaml: unmarshal errors:
  line 11: cannot unmarshal !!map into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-16670.yaml': yaml: unmarshal errors:
  line 7: cannot unmarshal !!seq into string
  line 12: cannot unmarshal !!map into string
[ERR] Could not parse file '/root/.local/nuclei-templates/cves/2018/CVE-2018-7251.yaml': yaml: unmarshal errors:
  line 9: cannot unmarshal !!map into string
  line 15: cannot unmarshal !!seq into string

**Comment by ehsandeep** [comment, marked as answer]
@qusaialhaddad update your nuclei to latest version, this error is due to use of oudated version of nuclei.
